lib/messagebird/call_list.rb

```diff
@@ -0,0 +1,9 @@
+require 'messagebird/list'
+
+class CallList < List
+
+  def data=(value)
+    @items = value.map { |i| @type.new i }
```

Comment:
What do you think about propagating this to `List`'s `items=` method and documenting why things are this way?

Reply:
done!